src/Api/GuestCarts.php

```diff
@@ -198,17 +198,25 @@ public function removeItem($cartId, $itemId)
      * @param string $customerEmail
      * @param string $customerFirstName
      * @param string $customerLastName
+     * @param $saleRep
+     * @param $salesLocation
+     * @param string $salesChannel
+     * @param string $orderType
      * @return Response
      * @throws Exception
      */
-    public function quoteLink($cartId, $adminEmail, $customerEmail, $customerFirstName, $customerLastName): Response
+    public function quoteLink($cartId, $adminEmail, $customerEmail, $customerFirstName, $customerLastName, $saleRep, $salesLocation, string $salesChannel = "Email", string $orderType = 'standard'): Response
```

Comment:
@developh 
Why did you only add types for some of the new arguments rather than all of them?